chapters/01-syllabus.Rmd

```diff
@@ -1,181 +1,152 @@
-
-## Learning outcomes
-
-- **Learning outcome/goal**: Ultimately, to be able to create an R package, that:
-    - Passes checks and requirements to (potentially) be submitted to CRAN
-    - Incorporates best practices in package development 
-    (unit testing, continuous integration, version control)
-    - Has code that is written and developed in a readable, clean, and
-    style-guide consistent format, so others can more easily read it
-    - Is designed to involve the community, such as gaining potential contributors
-    (including licensing, code of conduct, contributing guidelines, 
-    well described README)
-    - Has user-friendly (i.e. targeted to a novice to intermediate user)
-    documentation and tutorials on how to use the package (e.g. a nice website)
-    - Is adequately structured to make package management and maintenance easier
-    (e.g. with Issue Templates, Project Boards, Labels, contributing guidelines)
-
-## Who are these lessons for? {#rse-intro-personas}
+Computational skills are essential in the modern research environment, which
+increasingly have massive datasets or use high performance computing servers,
+along with a trend for a greater demand that scientific studies be
+reproducible and open. This book will teach you the tools, skills, and knowledge
+necessary to make research software that is open, reproducible, sustainable, and
+more rigorous. (TODO: other word?) First and foremost, the main learning goal of
+this course is that you learn to **create and develop an R package**.
+More specifically, the learning objectives are to:
+
+- Explain what an R package is, how it is structured, and how to set one up.
+- List and describe what some best (or good enough) practices are in package
+development and how to implement them. 
+- Explain what a code style guide is, why it is important, and how to write code
+that adheres to the style guide.
+- Describe why user-friendly and well-written documentation is crucial to
+building a usable package and some steps to take to achieve this type of
+documentation.
+- Cover what CRAN is, what policies and quality-control steps it has for packages,
+and how to prepare a package for submission to CRAN so that it passes these checks
+and requirements.
+
+Aside from the direct technical aspects of building an R package, the course
+will also cover topics that enhance and improve the process of package development.
+These learning objectives are to:
+
+- Describe and apply effective workflows and project management tools that help
+to coordinate the development of software, particularly for software that is
+complex or when working in a team.
+- Outline (and optionally practice) how to work productively in a small team,
+that is inclusive and welcoming.
+- Explain the importance of and learning to design software and documentation
+that involves the community, such as by gaining potential contributors.
+
+TODO: Include objectives about having ORCID and getting DOIs?
```

Comment:
I'm not personally in favor of creating an objective for ORCID and DOIs, as that will require us to go through the process ourselves. I think it *should* be included in the narrative, but as a discussion point as another option.